actionpack/lib/action_controller/log_subscriber.rb

```diff
@@ -60,9 +60,9 @@ def unpermitted_parameters(event)
       class_eval <<-METHOD, __FILE__, __LINE__ + 1
         def #{method}(event)
           return unless logger.info? && ActionController::Base.enable_fragment_cache_logging
-          key_or_path = event.payload[:key] || event.payload[:path]
+          key         = ActiveSupport::Cache.expand_cache_key(event.payload[:key] || event.payload[:path])
```

Comment:
`RAILS_CACHE_ID` var is weird: currently it only applies to caches made through actionpack, but not directly at `Rails.cache`.

The current problem with that can be here because you add `RAILS_CACHE_ID` to `payload[:key]` explicitly now and then expand it with same ENV var prefix again.


```
$ RAILS_CACHE_ID=22 rails c
Loading development environment (Rails 5.0.2)
development >> ActiveSupport::Cache.expand_cache_key('qq')
=> "22/qq/1"
development >> Rails.cache.fetch("qq") { 1}
=> 1
development >> Rails.cache.read(ActiveSupport::Cache.expand_cache_key('qq'))
=> nil
```
I think it is a bug and all direct calls to `Rails.cache` should also incorporate `RAILS_CACHE_ID`.